src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerTimerTask.java

```diff
@@ -63,6 +63,9 @@ public void run() {
         if (StringUtils.isEmpty(trigger.getTriggerConfigURL())) {
             return;
         }
+        if (!trigger.getJob().isBuildable()) {
```

Comment:
To be safe, should check `gerritTrigger.getJob() != null` first.

Reply:
Fixed. Thx